api/CustomElementRegistry.json

```diff
@@ -0,0 +1,66 @@
+{
+  "api": {
+    "CustomElementRegistry": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomElementRegistry",
+        "support": {
+          "webview_android": {
+            "version_added": "56",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+          "chrome": {
+            "version_added": "54",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+          "chrome_android": {
+            "version_added": "61",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+          "edge": {
+            "version_added": false,
+            "notes": "Under consideration"
+          },
+          "edge_mobile": {
+            "version_added": false,
+            "notes": "Under consideration"
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "Enabled through the `dom.webcomponents.enabled` preference in `about:config`",
```

Comment:
This note is not needed, the MDN compat tables generate a consistent note from the flag object which we can also localize in the future. Please remove.

Reply:
fyi, the generated note will be:

"This feature is behind the `dom.webcomponents.enabled` preference (needs to be set to `true`). To change preferences in Firefox, visit about:config."

Reply:
@Elchi3 pardon my inexperience this schema is super confusing at least to me. Why would that not be a note of the flags section?

Reply:
The flag section contains all the information about the pref to enable. There is no need to write the note explicitely. From the flag section information, macros can generate the note synthetically.

Reply:
Ah I see. Great explanation @teoli2003.

Reply:
@teoli2003 Fixed in @teoli2003 Fixed in https://github.com/mdn/browser-compat-data/pull/623/commits/0b27628b425fa7ed4d1b2447ac839033a4960c7b